es-model-archive/kf-es-model-v0.5.0_1.2.0/file_centric.mapping.json

```diff
@@ -0,0 +1,528 @@
+                "kf_id": {
+                  "type": "keyword"
+                }, 
+                "ncit_id_anatomical_site": {
+                  "type": "keyword"
+                }, 
+                "ncit_id_tissue_type": {
+                  "type": "keyword"
+                }, 
+                "sequencing_center_id": {
+                  "type": "keyword"
+                }, 
+                "shipment_date": {
+                  "type": "keyword"
+                }, 
+                "shipment_origin": {
+                  "type": "keyword"
+                }, 
+                "source_text_anatomical_site": {
+                  "type": "keyword"
+                }, 
+                "source_text_tissue_type": {
+                  "type": "keyword"
+                }, 
+                "source_text_tumor_descriptor": {
+                  "type": "keyword"
+                }, 
+                "spatial_descriptor": {
+                  "type": "keyword"
+                }, 
+                "uberon_id_anatomical_site": {
+                  "type": "keyword"
+                }, 
+                "volume_ml": {
+                  "type": "float"
+                }
+              }, 
+              "type": "nested"
+            }, 
+            "consent_type": {
+              "type": "keyword"
+            }, 
+            "diagnoses": {
+              "properties": {
+                "age_at_event_days": {
+                  "type": "integer"
+                }, 
+                "diagnosis": {
+                  "type": "keyword"
+                }, 
+                "diagnosis_category": {
+                  "type": "keyword"
+                }, 
+                "external_id": {
+                  "type": "keyword"
+                }, 
+                "icd_id_diagnosis": {
+                  "type": "keyword"
+                }, 
+                "kf_id": {
+                  "type": "keyword"
+                }, 
+                "mondo_id_diagnosis": {
+                  "type": "keyword"
+                }, 
+                "ncit_id_diagnosis": {
+                  "type": "keyword"
+                }, 
+                "source_text_diagnosis": {
+                  "type": "keyword"
+                }, 
+                "source_text_tumor_location": {
+                  "type": "keyword"
+                }, 
+                "spatial_descriptor": {
+                  "type": "keyword"
+                }, 
+                "uberon_id_tumor_location": {
+                  "type": "keyword"
+                }
+              }, 
+              "type": "nested"
+            }, 
+            "diagnosis_category": {
+              "type": "keyword"
+            }, 
+            "ethnicity": {
+              "type": "keyword"
+            }, 
+            "external_id": {
+              "type": "keyword"
+            }, 
+            "family": {
+              "properties": {
+                "available_data_types": {
```

Comment:
Are fields directly under `family` supposed to be taken from `dataservice` api? If that's the intention, there are a few fields need to be removed from `es_mapper.yaml`, such as: `available_data_types`, `shared_hpo_ids`, `father_id` and `mother_id`. These fields were added by the ETL previously.

If some of these fields are still intended to be populated by the ETL, then keep those.

Reply:
We had not discussed removing those so I left them in to continue to be populated by ETL - for example `father_id` and `mother_id` are in use in the reports.